theta_sketch_agg_union.sql

```diff
@@ -15,8 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 
-CREATE OR REPLACE AGGREGATE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_agg_union(sketch BYTES, lg_k INT64 NOT AGGREGATE) RETURNS BYTES LANGUAGE js
-OPTIONS (library=["$GCS_BUCKET/theta_sketch.mjs"]) AS R"""
+CREATE OR REPLACE AGGREGATE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_agg_union(sketch BYTES, lg_k INT64 NOT AGGREGATE)
```

Comment:
This is the only one that didn't require a seed.  Is this correct?

Reply:
not implemented yet. I will do that shortly